src/toxcore/binary_io.rs

```diff
@@ -71,13 +71,13 @@ macro_rules! parse_error {
 pub type ParseResult<'a, Output> = Result<Parsed<'a, Output>, ParseError>;
 
 /// De-serialize from bytes, or return `None` if de-serialization failed.
```

Comment:
This doc needs to be adjusted – perhaps to `Trait for de-serializing from bytes`, or something..